examples/ecs/fallible_systems.rs

```diff
@@ -1,18 +1,29 @@
-//! Showcases how fallible systems can be make use of rust's powerful result handling syntax.
+//! Showcases how fallible systems and observers can make use of rust's powerful result handling
```

Comment:
```suggestion
//! Showcases how fallible systems and observers can make use of Rust's powerful result handling
```
Not your fault, but while we're here...

Reply:
Resolved in [1c4b6e5](https://github.com/bevyengine/bevy/pull/17731/commits/1c4b6e5ed1420f3efc32018e1a890e396c5225b2).